src/sonic.c

```diff
@@ -319,6 +319,28 @@ XML_parser_general(void *data, const char *elem, const char **attr)
     LinkTable_add(linktbl, link);
 }
 
+static void sanitize_LinkTable(LinkTable *linktbl) {
+    for (int i = 0; i < linktbl->num; i++) {
+	    if (strcmp(linktbl->links[i]->linkname, ".") == 0) {
+		    strcpy(linktbl->links[i]->linkname, "dot");
```

Comment:
I don't really like this sanitation scheme, one single character gets turned into 3.

Reply:
To be honest I just applied [what you suggested in this issue](https://github.com/fangfufu/httpdirfs/issues/56#issuecomment-703016710). I wanted to turn it into a single character but the problem is that the other character I use could already by used in the index, for example if I rename it `-` and there is an artist named `-artist`.